multimodal/src/autogluon/multimodal/configs/pretrainer/default.yaml

```diff
@@ -0,0 +1,9 @@
+pretrainer:
```

Comment:
Can we add this pretraining config here? (https://github.com/autogluon/autogluon/tree/master/multimodal/src/autogluon/multimodal/configs/pretrain) The `pretrain` folder was added to store all the pretraining configs.